scripts/run_backend_tests.py

```diff
@@ -97,13 +98,13 @@
     description="""
 Run this script from the oppia root folder:
     python -m scripts.run_backend_tests
-IMPORTANT: Only one of --test_path,  --test_target, and --test_shard
-should be specified.
+IMPORTANT: Only one of --test_path, --test_targets, --run_on_changed_files,
+and --test_shard should be specified.
 """)
 
 _EXCLUSIVE_GROUP: Final = _PARSER.add_mutually_exclusive_group()
 _EXCLUSIVE_GROUP.add_argument(
-    '--test_target',
+    '--test_targets',
```

Comment:
Generally, we should deprecate the old flag and add a new one but considering there are very few developers using it so I'm fine with changing it directly and updating developers over the email. Do we need to update any wiki page with this change? Also, we will have to write out on oppia-dev explaining these changes (in case we are directly replacing it!

Reply:
Yes we have to change the https://github.com/oppia/oppia/wiki/Backend-tests which I will do after this PR most likely and I am planning to write an email about changes to the tests scripts and also about the pre-push hook after this PR.